app/components/views/Buy/index.module.css

```diff
@@ -0,0 +1,25 @@
+@value bp-medium from "@klimadao/lib/theme/breakpoints.css";
+@value textButton from "@klimadao/lib/theme/common.module.css";
```

Comment:
I'm totally cool merging this `.module.css`--- but at some point need to put the hammer down and say "only `emotion` from now on" :P

👉 #10

Reply:
I'll just take it out. I thought I had to do it, but I half-assed it anyway.